### PR TITLE
3824: Content updates

### PR DESCRIPTION
**Pull request by alistairknight**

### CODE REVIEW

File: grc/templates/birth-registration/dob.html [hunk 1]
```diff
@@ -19,17 +19,13 @@
...
                         <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                             <h1 class="govuk-fieldset__heading">
                                 What is the date of birth on your birth or adoption certificate?
                             </h1>
                         </legend>
 
-                        <div class="govuk-hint" id="fieldset-hint">
-                            Enter the year in full - for example, 1987
-                        </div>
-
```
Comment (jamesgriff) [inline grc/templates/birth-registration/dob.html:32]: Have we been asked to remove this?
I wonder if we discussed it, but forgot to add it to the Google Doc.
Can you check with Spence or Alex to see if they want to keep this?

Comment (alistairknight) [reply grc/templates/birth-registration/dob.html:32]: Tor confirmed it should stay

File: grc/templates/submit-and-pay/confirmation.html [hunk 2]
```diff
@@ -23,25 +23,27 @@ <h2 class="govuk-heading-m">
...
                 {% if application_data.submit_and_pay_data.is_using_ex160_form %}
                     <li>
                         an
                         <a href="https://www.gov.uk/government/publications/apply-for-help-with-court-and-tribunal-fees"
                            rel="external" target="_blank" class="govuk-link">
-                            EX160 form (opens in new tab)
+                            EX160 form (opens in a new tab)
                         </a>
                     </li>
                 {% endif %}
```
[the remark this answers is not in the document]
Comment (alistairknight) [reply grc/templates/submit-and-pay/confirmation.html]: Spoke to Spence, content has been updated

File: grc/templates/birth-registration/dob.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
                 {{ form.csrf_token }}
 
                 <div class="govuk-form-group {{ 'govuk-form-group--error' if form.errors }}">
-                    <fieldset class="govuk-fieldset" role="group" aria-describedby="fieldset-hint">
```
Comment (jamesgriff) [inline grc/templates/birth-registration/dob.html:22]: Since we're keeping the hint, can you also keep this `aria-describedby`?

**Review by jamesgriff — APPROVED** [review]

**Pull request merged**